core/src/main/java/org/bitcoinj/quorums/GetQuorumRotationInfo.java

```diff
@@ -50,7 +48,6 @@ public GetQuorumRotationInfo(NetworkParameters params, long baseBlockHashesCount
 
     @Override
     protected void parse() throws ProtocolException {
-        baseBlockHashesCount = readUint32();
         int count = (int)readVarInt();
         baseBlockHashes = new ArrayList<>(count);
         for (int i = 0; i < count; ++i) {
```

Comment:
as seen here, the number of base block hashes included in this message was previously included twice (once as a Uint32 and then as a VarInt.   VarInt, also called compactSize, is the way Bitcoin stores integers using the smallest size possible.  sometimes 1 byte, or 2 bytes, or 3 or 4).